src/sentry/tasks/releasemonitor.py

```diff
@@ -0,0 +1,178 @@
+import logging
+import time
+from datetime import datetime, timedelta
+
+from django.db.models import Q, Subquery
+from django.utils import timezone
+
+# from snuba_sdk import Condition, Column, Direction, Entity, Granularity, Op, OrderBy, Query
+from snuba_sdk.conditions import Condition, Op
+from snuba_sdk.expressions import Granularity
+from snuba_sdk.orderby import Direction, OrderBy
+from snuba_sdk.query import Column, Entity, Query
+
+from sentry.models import Release, ReleaseProjectEnvironment
+from sentry.tasks.base import instrumented_task
+from sentry.utils import metrics, snuba
+
+REQUIRED_ADOPTION_PERCENT = 0.1
+CHUNK_SIZE = 1000
+MAX_SECONDS = 60
+
+logger = logging.getLogger("tasks.releasemonitor")
+
+
+@instrumented_task(
+    name="sentry.tasks.monitor_release_adoption",
+    queue="releasemonitor.monitor_release_adoption",
```

Comment:
Same here, probably just `releasemonitor`. I think these should just share a queue (unless someone advised otherwise)